examples/common/python/verify_report/verify_attestation_report.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import logging
+import verify_report.verify_report as verify_report_util
+
+logger = logging.getLogger(__name__)
+
+def verify_attestation_report(enclave_info):
+    verify_status = 0
+    verification_report = json.dumps(enclave_info["proof_data"]["verification_report"])
+    ias_report_cert = enclave_info["proof_data"]["ias_report_signing_certificate"]
+    proof_signature = enclave_info["proof_data"]["ias_report_signature"]
+
+    quote_status = verify_report_util.verify_quote(verification_report, 1)
```

Comment:
1 corresponds to enclave quote status "group_out_of_date_is_ok". 
If passed a value equal to 1, then the function doesn't throw error if quoteStatus is group-out-of-date

Reply:
It's not clear from function interface. Do you want to expose this through function interface ? Are we passsing any other value other than 1. Else avoid this parameter and can hide these details

Reply:
I added a comment for the value 1. We need to expose this option to client, so that client can decide if the quote status "group-out-of-date" is okay.

Reply:
If we want to expose then let's define enum for this in python and do the conversion inside the function to map to C++ datatype

Reply:
Ok, i defined an enum for QuoteStatus.
To simplify, i would just pass value of the enum as function param so that a seperate function to map to C++ datatype is avoided. 
Eg: quote_status = verify_report_util.verify_quote(verification_report, QuoteStatus.GROUP_OUT_OF_DATE_OK.value)

Hope this is fine.